core/Cargo.toml

```diff
@@ -22,6 +22,7 @@ cli = []
 dex = []
 dev-telemetry = ["telemetry", "iroha_telemetry/dev-telemetry"]
 expensive-telemetry = ["iroha_telemetry/metric-instrumentation"]
+debug-wasm = []
```

Comment:
```suggestion
```

this is so small feature that it's ok to include it always. The smartcontract will choose whether or not it will link against it